### PR TITLE
Bugs 160324 vk home collection page rework

### PR DESCRIPTION
**Pull request by VijayMarmetian**
Pull Request (PR) Request Template:

### Problem
To built a brand collection page and modify the home page design 

Monday Card Link: 
https://marmeto-force.monday.com/boards/5752050817/pulses/6271459014

### Solution
Built a new section and modified existing sections for homepage and collection page.

<Visual representation or screenshots>

### Impacted Areas
<Explain other areas of the application that could be affected by this change.>

###Preview link with store password
https://parkavenue-india.myshopify.com/?preview_theme_id=166093488408
https://parkavenue-india.myshopify.com/pages/collections?preview_theme_id=166093488408
marmeto

### Done Checklist
- [x] Performed unit testing to ensure functionality and identify potential issues.
- [x] Followed basic code standards and best practices. (indentation, media queries, basic customizer things)
- [x] Added comments to explain complex logic or improve code readability.
- [x] Followed Naming conventions and proper variable declaration.
- [x] Handled edge cases and errors gracefully to prevent crashes.
- [x] Optimized the code for production to ensure performance.

Please review this PR and provide feedback. Thank you!

### CODE REVIEW

File: sections/brand-collection.liquid [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+      ],
+      "blocks": [
+        {
+          "type": "slide",
+          "name": "Slide",
+          "settings": [
+            {
+              "type": "image_picker",
+              "label": "Image (mobile)",
+              "id": "image_small"
```
Comment (arshiya-khanam) [inline sections/brand-collection.liquid:445]: add recommended size

File: sections/brand-collection.liquid [hunk 2]
```diff
@@ -0,0 +1,484 @@
...
+            },
+            {
+              "type": "image_picker",
+              "label": "Image (desktop)",
+              "id": "image_desktop"
```
Comment (arshiya-khanam) [inline sections/brand-collection.liquid:450]: add recommended size

File: sections/contact-image-banner.liquid [hunk 3]
```diff
@@ -7,7 +7,7 @@
    
      {% if section.settings.show_heading %}
     <div class="contact-heading">
-        <span class="h2"> HOW CAN</span><span class="svg-container">{% render "avenue-icon" %} </span><span class="h2">WE HELP ?</span>
+        <span class="h2"> HOW CAN WE HELP ?</span>
```
Comment (arshiya-khanam) [inline sections/contact-image-banner.liquid:10]: do not hard code

File: sections/image-banner-carousel.liquid [hunk 4]
```diff
@@ -188,9 +191,14 @@
             "label": "Image (desktop)",
             "id": "image_desktop"
```
Comment (arshiya-khanam) [inline sections/image-banner-carousel.liquid:192]: add recommended size

File: sections/video-section.liquid [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+        {
+          "type": "video",
+          "id": "mobile_video",
+          "label": "Select mobile video",
+          "info": "If not selected desktop video will be shown"
+        },
+        {
+          "type": "image_picker",
+          "id": "cover_image",
+          "label": "t:sections.video.settings.cover_image.label"
```
Comment (arshiya-khanam) [inline sections/video-section.liquid:142]: add recommended size

**Review by arshiya-khanam — CHANGES_REQUESTED** [review]

File: assets/custom.css [hunk 6]
```diff
@@ -339,3 +336,20 @@ a {
 	justify-content: flex-end;
 	align-items: flex-end;
 }
+.ibc_image_container{
+	position: relative;
+}
+.ibc_image_container .overlay-button{
+	position: absolute;
```
Comment (Manuraj-Marmeto) [inline assets/custom.css:343]: the intentation is bad here 
Please make it aligned

**Review by Manuraj-Marmeto — CHANGES_REQUESTED** [review]

**Pull request merged**